regression/cbmc/ACSL/fail-operators.c

```diff
@@ -0,0 +1,36 @@
+void boolean()
+{
+  __CPROVER_bool a, b;
+  __CPROVER_assert(!(a ≡ b) == (a == b), "≥");
+  __CPROVER_assert(!(a ≢ b) == (a != b), "≢");
+  __CPROVER_assert(!(a ⇒ b) == (!a || b), "⇒");
```

Comment:
These looked a bit suspicious to me, but it looks like the [C Compiler](https://gcc.gnu.org/onlinedocs/gcc-4.1.2/cpp/Character-sets.html) (or at least, GCC does) determines file encoding from reading the file, defaulting to either ASCII or UTF-8, and we are using `ifstream` to read files, which has its encoding information set through `std::locale`, so it looks like this should work.